packages/devextreme/js/ui/autocomplete.d.ts

```diff
@@ -162,6 +163,20 @@ export type OptionChangedEvent = EventInfo<dxAutocomplete> & ChangedOptionInfo;
  */
 export type PasteEvent = NativeEventInfo<dxAutocomplete, ClipboardEvent>;
 
+/**
+ * @docid _ui_autocomplete_SelectionChangingEvent
+ * @public
+ * @type object
+ * @inherits EventInfo,_ui_drop_down_editor_ui_drop_down_list_SelectionChangingInfo
+ */
+export type SelectionChangingEvent = EventInfo<dxAutocomplete> & SelectionChangingInfo & {
+    /**
+     * @docid _ui_autocomplete_SelectionChangingEvent.cancel
+     * @type Boolean|Promise<Boolean>
+     */
+     cancel: boolean | PromiseLike<boolean>;
```

Comment:
No, it doesn't matter because it will be presented as a table https://js.devexpress.com/jQuery/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#onShowing